### ISSUE TITLE
[0.8 Preview] Building class library with XAML controls in Release config fails

### ISSUE THREAD

**Issue by andrewleader** [microsoft/ProjectReunion#923]
**Describe the bug**
This is a **known issue** that affects the **0.8 Preview**. If you have a WinUI 3 class library that defines XAML controls and are using the 0.8 Preview, attempting to build your app in **Release** config will fail with the error seen below. The stable 0.5 release is not affected by this issue.

```
Error    MSB3030
Could not copy the file
"C:\Users\aleader\source\repos\App24Experimental\ClassLibrary1\bin\x86\Release\net5.0-windows10.0.19041.0\UserControl1.xbf"
because it was not found.     ClassLibrary1 C:\Users\aleader\.nuget\packages\microsoft.projectreunion.foundation\0.8.0-preview\build\Microsoft.ApplicationModel.Resources.PriGen.targets
```

**Workaround**
Building in Debug config should work. If you need to build in Release config, there is no known workaround (aside from switching back to 0.5 stable release).

We've fixed this with pull request #917 but there is not yet a new version available with this fix.

**Steps to reproduce the bug**

1. Create a 0.8 Preview WinUI 3 desktop app
2. Add a 0.8 Preview WinUI 3 class library
3. Add a new UserControl (which uses a XAML page)
4. Reference the class library from the desktop app
5. In your desktop app, use the UserControl you defined
6. Attempt to build the app in Release config

**Expected behavior**
App should build successfully.

**Version Info**

NuGet package version: 
`0.8.0-preview`

**Comment by andrewleader** [comment]
Closing as this has been fixed with pull request #917, but note that no release with the fix is available yet.

**Comment by huichen123** [comment]
a possible workaround is to add "<GenerateLibraryLayout>true</GenerateLibraryLayout>" in csproj.
